src/app/shared/components/external-cluster-credentials/provider/eks/component.ts

```diff
@@ -71,10 +78,25 @@ export class EKSCredentialsComponent implements OnInit, OnDestroy {
     this._unsubscribe.complete();
   }
 
+  private _getEKSRegions(): void {
+    const accessKeyID = this.form.get(Controls.AccessKeyID).value;
+    const secretAccessKey = this.form.get(Controls.SecretAccessKey).value;
+    if (accessKeyID && secretAccessKey) {
+      this._externalClusterService
+        .getEKSRegions(accessKeyID, secretAccessKey)
+        .subscribe((regions: string[]) => (this.regions = regions));
+      this.form.get(Controls.Region).setValue('eu-central-1');
```

Comment:
I think this should be moved inside `subscribe` and then we should add a check there that `eu-central-1` exists in `this.regions`. Perhaps it will better we can create a constant for `eu-central-1` similar to how you created for location in AKS component.